docs/services/matterbridge.md

```diff
@@ -0,0 +1,23 @@
+# Matterbridge
+
+[Matterbridge](https://github.com/42wim/matterbridge)
+
+## Configuration
+
```

Comment:
Perhaps we should point people to https://github.com/42wim/matterbridge/wiki/How-to-create-your-config for tips on how they can generate the Matterbridge configuration.

Reply:
Done :slightly_smiling_face: